cron/checkmail2.php

```diff
@@ -127,20 +141,21 @@
     }
 
     $from = $header->from[0]->mailbox . "@" . $header->from[0]->host;
+    $subject = utf8_decode_email_subject($header->subject);
```

Comment:
I would recommend using `imap_utf8($header->subject)` instead to correctly handle the decoding. It will support correctly converting from any encoding like ISO-8859-1, etc.

Reply:
Indeed it's best